tests/unit/nupic/support/decorators_test.py

```diff
@@ -22,18 +22,251 @@
 
 """Unit tests for the nupic.support.decorators module.
 
-TODO: add tests for logEntryExit and retry decorators
+TODO: add tests for logEntryExit
 """
 
 
+from mock import patch, call, Mock
 
-from mock import patch, Mock
+from nupic.support.unittesthelpers.testcasebase import unittest
+from nupic.support import decorators
 
 
-from nupic.support.unittesthelpers.testcasebase import unittest
 
+class TestParentException(Exception):
+  pass
 
```

Comment:
There should be 3 lines between global-level constructs (e.g., class definitions, functions)